File: ECommerceSite.Client/src/app/shared/models/Games.ts
import { Tag } from "./Tag";

export class Games{
  gameId!:number; 
  gameName!:string;
  gamePrice!:number;
  tags?:Tag[];
  favorite?:boolean = false;
  gameImageUrl!:string;
  releaseDate!:string;
  description?:string;
}

File: ECommerceSite/ECommerceSite/Games.cs
using System.ComponentModel.DataAnnotations;

namespace ECommerceSite
{
  public class Games
  {
    [Key]
    public int gameId { get; set;  }
    public string gameName { get; set; }
    public double gamePrice { get; set; }
    public string gameImageUrl { get; set; }
    public string? gameDescription { get; set; }
    public DateTime gameReleaseDate { get; set; }
  }
}
